.github/workflows/ci.yml

```diff
@@ -108,7 +108,7 @@ jobs:
           if-no-files-found: error
           retention-days: 1
       - name: Test
-        run: ./mill -i java-class-name-tests.mostly-static.test
+        run: ./mill -i  --no-filesystem-checker java-class-name-tests.mostly-static
```

Comment:
similarly to `static` tests, we do IO ops which are not allowed under Mill 1.0.0 unless we pass this flag.